locales-pending/premium.ftl

```diff
@@ -230,39 +230,69 @@ dashboard-top-banner-monitor-protects-your-even-more-learn-more = Learn more
 dashboard-top-banner-no-exposures-found-title = No exposures found
 # Variables:
 # $data_broker_sites_total_num is the total number of data broker sites available to scan. This will always be plural.
-dashboard-top-banner-no-exposures-found-description = Great news! We searched all known data breaches and { $data_broker_sites_total_num } data broker sites that sell personal info and found no exposures. Upgrade to premium and we’ll monitor for any new exposures.
+dashboard-top-banner-no-exposures-found-description = Great news! We searched all known data breaches and { $data_broker_sites_total_num } data broker sites that sell personal info and found no exposures.
+dashboard-top-banner-no-exposures-found-upsell-info = Upgrade to premium and we’ll monitor for any new exposures.
```

Comment:
```suggestion
dashboard-top-banner-no-exposures-found-upsell-info = Upgrade to { -brand-premium } and we’ll monitor for any new exposures.
```

I assume premium should be uppercase here.

Reply:
Yes, uppercase is correct!

Reply:
I think you may have forgotten to apply this suggestion? :)